ts/Accessibility/Components/SeriesComponent/SeriesDescriber.ts

```diff
@@ -597,14 +597,17 @@ function defaultSeriesDescriptionFormatter(
         ): (boolean|Axis) {
             return chart[coll] && chart[coll].length > 1 && series[coll];
         },
+        index = series.index + 1,
+        numSeries = chart.series && chart.series.length,
+        numPoints = series.points && series.points.length,
         xAxisInfo = getSeriesAxisDescriptionText(series, 'xAxis'),
         yAxisInfo = getSeriesAxisDescriptionText(series, 'yAxis'),
         summaryContext = {
             name: series.name || '',
-            ix: (series.index as any) + 1,
-            numSeries: chart.series && chart.series.length,
-            numPoints: series.points && series.points.length,
-            series: series
+            ix: index,
```

Comment:
In this `summaryContext` this property is called `ix`, while in the `descriptionFormat` context below the same property is called `index`. And in both cases it is `series.index + 1` which may itself be confusing. What about using [number](https://english.stackexchange.com/questions/366834/what-is-a-word-that-means-one-based-index) for the one-based index?

Reply:
Makes sense, will change!